tests/software/prokka/test.yml

```diff
@@ -0,0 +1,17 @@
+- name: prokka
+  command: nextflow run ./tests/software/prokka -entry test_prokka -c tests/config/nextflow.config
+  tags:
+    - prokka
+  files:
+    - path: output/prokka/test/test.gff
```

Comment:
Maybe you can add a checksum for at least one of them?

Reply:
will do, thanks for the review

Reply:
Or all of them?

Reply:
Added checksums for those files without a time stamp within